src/layouts/components/Header.tsx

```diff
@@ -62,6 +62,12 @@ export function Header(): JSX.Element {
     }
   }, [menu]);
 
+  useEffect(() => {
+    if (dimension.width >= 1024) {
```

Comment:
let's add `menu` check also, to decrease the ripple effect when setting new state

Reply:
updated !